### PR TITLE
Gitignore changes

### PR DESCRIPTION
**Pull request by jenniferhalgh**
Added the deleted folders "libraries" and "cmake-build-debug" to gitignore.

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [ ] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [ ] Acceptance criteria of the linked stories satisfied
- [ ] Code change comprehended by those who approved it
- [ ] State of target branch maintained or improved after the code change

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -68,3 +68,7 @@ google-services.json
 
 # Android Profiling
 *.hprof
+
+
+arduino/libraries/
+arduino/cmake-build-debug/
```
Comment (platisd) [inline .gitignore:74]: Maybe add these to #90 instead?

Comment (jenniferhalgh) [reply .gitignore:74]: I'm not sure how I'm supposed to add it. Should I close this pull request and push it again to pull request #90?

Comment (platisd) [reply .gitignore:74]: Checkout the branch that #90 is on, make the same changes, commit and push them

**Comment by jenniferhalgh** [comment]
This pull request is closed because it is handled in pull request #90.

**Pull request closed without merging**